benchmarks/Neo.VM.Benchmarks/OpCode/OpCode.Nop.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// OpCode.Nop.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using BenchmarkDotNet.Attributes;
+
+namespace Neo.VM.Benchmark.OpCode;
+
+public class OpCode_Nop : OpCodeBase
+{
+    [GlobalSetup]
+    public void Setup()
```

Comment:
You want to call the base Setup? or override?

Reply:
I have deleted unfinished benchmark, this pr focus on creating the benchmark framework itself.